maven-plugin/src/main/java/org/moditect/mavenplugin/add/AddModuleInfoMojo.java

```diff
@@ -117,7 +127,13 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     		getLog().debug("Mojo 'add-module-info' not executed on packaging type '"+project.getModel().getPackaging()+"'");
     		return;
     	}
-    	
+
+        if (exclusions != null) {
+            ArtifactFilter scopeFilter     = new ScopeArtifactFilter("compile+runtime");
```

Comment:
Revisit this as technically `provided` scope could map to `requires static` instructions in the module descriptor.